tests/integration_tests/performance/test_block_performance.py

```diff
@@ -79,7 +81,11 @@ def run_fio(env_id, basevm, mode, bs):
         .with_arg("--ioengine=libaio")
         .with_arg("--iodepth=32")
         .with_arg(f"--ramp_time={CONFIG['omit']}")
-        .with_arg(f"--numjobs={CONFIG['load_factor'] * basevm.vcpus_count}")
+        .with_arg(f"--numjobs={num_jobs}")
+        # Set affinity of the entire fio process to a set of vCPUs equal in size to number of workers
+        .with_arg(f"--cpus_allowed={','.join(str(i) for i in range(num_jobs))}")
+        # Instruct fio to pin one worker per vcpu
+        .with_arg("--cpus_allowed_policy=split")
```

Comment:
Have we tested how does this perform? I.e. does it pass the block perf test, or would we need to gather baselines?

Reply:
I don't expect it to change anything, but let me run a pipeline anyway

Reply:
Interesting, it does make m5d.metal fail. Well, I'll combine this change with the rest of my changes that require new baselines then and just bite the bullet and recollect